menus/track.py

```diff
@@ -261,19 +270,21 @@ def sustain_mod(self):
             asm.RTS(),
 
             "SUSTAIN_SCROLL_AREA",
-            asm.LDA(0x0d, asm.DIR),
-            asm.BIT(0x80, asm.IMM8),    # b pressed?
-            asm.BNE("EXIT_SCROLL_AREA"),
+            asm.LDA(0x09, asm.DIR),
+            asm.BIT(0x80, asm.IMM8),     # b pressed?
+            asm.BNE("EXIT_SCROLL_AREA"), # branch if so
+        ]
+
+        src.extend(self.common.get_flags_a_check_src(self.common.invoke_flags_submenu[submenu_idx]))
```

Comment:
### Change 3: update usage of get_submenu_src similar to pregame.py
```suggestion
        for submenu_idx in self.common.flags.submenus.keys():
            src.extend(self.common.get_submenu_src(submenu_idx, self.common.invoke_flags_submenu[submenu_idx]))
```

Reply:
Thanks, I incorporated this change.